tests/schedule_test.go

```diff
@@ -224,8 +224,8 @@ func (s *ScheduleFunctionalSuite) TestBasics() {
 	s.NoError(err)
 
 	// sleep until we see two runs, plus a bit more to ensure that the second run has completed
-	s.Eventually(func() bool { return atomic.LoadInt32(&runs) == 2 }, 12*time.Second, 500*time.Millisecond)
-	time.Sleep(1 * time.Second) //nolint:forbidigo
+	s.Eventually(func() bool { return atomic.LoadInt32(&runs) == 2 }, 15*time.Second, 500*time.Millisecond)
```

Comment:
For these `Eventually` assertions with loose timeout bounds, what do you think about having a constant we can tweak more centrally (either to the suite, or to all functional tests)? Doesn't have to be part of this PR, ofc.

Reply:
I dunno, the "right" value for a timeout does seem kind of contextual. E.g. we can say we always wait 10 seconds for something to appear in visibility... but the getScheduleEntryFomVisibility below doesn't just wait for visibility, it also waits for something test-dependent in the data it gets. So even that pretty straightforward one isn't so obvious.

Worth discussing with the testing crew, though.